basic-basic-crud-functions.ipynb

```diff
@@ -163,8 +163,8 @@
    "outputs": [],
    "source": [
     "with f:\n",
-    "    f.search(docs[0], top_k=3,\n",
-    "             on_done=lambda x: [print(m.dict()) for m in x.docs[0].matches])"
+    "    f.search(docs[0], parameters={'top_k': 4},,\n",
+    "             on_done=lambda x: [print(m.dict()) for m in x.data.docs[0].matches])"
```

Comment:
I think without `data` should work, can u check?

Reply:
For me this notebook fails:

```python
           pod0@70478[C]:can not load the executor from _index
           pod0@70478[E]:ExecutorFailToLoad() during <class 'jina.peapods.runtimes.zmq.zed.ZEDRuntime'> initialization
 add "--quiet-error" to suppress the exception details
Traceback (most recent call last):
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/peapods/runtimes/zmq/zed.py", line 91, in _load_executor
    runtime_args=vars(self.args),
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/jaml/__init__.py", line 553, in load_config
    return JAML.load(tag_yml, substitute=False)
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/jaml/__init__.py", line 89, in load
    r = yaml.load(stream, Loader=JinaLoader)
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/yaml/constructor.py", line 51, in get_single_data
    return self.construct_document(node)
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/yaml/constructor.py", line 55, in construct_document
    data = self.construct_object(node)
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/yaml/constructor.py", line 100, in construct_object
    data = constructor(self, node)
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/yaml/constructor.py", line 429, in construct_undefined
    node.start_mark)
yaml.constructor.ConstructorError: could not determine a constructor for the tag '!_index'
  in "<unicode string>", line 1, column 1:
    !_index
    ^

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/peapods/peas/__init__.py", line 152, in run
    runtime = self._build_runtime()
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/peapods/peas/__init__.py", line 138, in _build_runtime
    timeout_ctrl=self._timeout_ctrl,
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/peapods/runtimes/zmq/zed.py", line 59, in __init__
    self._load_executor()
  File "/home/cristian/envs/notebooks/lib/python3.7/site-packages/jina/peapods/runtimes/zmq/zed.py", line 104, in _load_executor
    raise ExecutorFailToLoad from ex
```

Reply:
replaced 1.x _index builtin indexer with NumpyPostgresSearcher

Reply:
> I think without `data` should work, can u check?

yes it works